src/main/java/org/kainos/ea/db/ResponsibilityDao.java

```diff
@@ -16,29 +16,26 @@ public class ResponsibilityDao {
     private DatabaseConnector databaseConnector = new DatabaseConnector();
 
     public List<Responsibility> getRoleResponsibilityById(int id) throws SQLException {
+
+        Connection c = databaseConnector.getConnection();
+
         String query = "SELECT rt.responsibilityName FROM Responsibility rt "
                 + "JOIN JobRole jr ON rt.roleID = jr.roleID WHERE jr.roleID = ?";
 
-        List<Responsibility> responsibilitiesList = new ArrayList<>();
-
-        try (Connection c = databaseConnector.getConnection();
-             PreparedStatement ps = c.prepareStatement(query)) {
+        PreparedStatement ps = c.prepareStatement(query);
+        ps.setInt(1, id);
 
-            ps.setInt(1, id);
+        ResultSet rs = ps.executeQuery();
 
-            try (ResultSet rs = ps.executeQuery()) {
-                while (rs.next()) {
-                    Responsibility responsibility = new Responsibility();
-                    responsibility.setResponsibilityName(rs.getString("responsibilityName"));
-                    responsibilitiesList.add(responsibility);
-                }
-            }
+        List<Responsibility> responsibilitiesList = new ArrayList<>();
```

Comment:
There should only ever one record back from the SQL query, you should just use a single Responsibility object.